Tests/AsyncHTTPClientTests/HTTPClientTests.swift

```diff
@@ -333,6 +333,38 @@ class HTTPClientTests: XCTestCase {
         XCTAssertEqual(.ok, response.status)
     }
 
+    func testGetHttpsWithIPv6() throws {
+        let localHTTPBin = HTTPBin(.http1_1(ssl: true), bindTarget: .localhostIPv6RandomPort)
```

Comment:
These tests need to preflight that IPv6 loopback is available. Some systems do not configure the IPv6 module in Linux and so lack the ability to run these tests.

Usually the best way to do this is to just try binding and connecting to a random IPv6 port. We can hide that behind a `static let`.

Reply:
I've added something - not sure if that's quite what you meant. I've added a NIOTS check to it, because the ServerBootstrap doesn't like the default ELG otherwise, and I figure we can rely on Network.framwork to support IPv6 (like... it should be an error if it doesn't work).